web/styles/subscriptions.css

```diff
@@ -288,13 +288,14 @@ h4.user_group_setting_subsection_title {
     padding: 2px 10px;
 
     cursor: pointer;
+    visibility: hidden;
```

Comment:
Moved here from below due to `Unexpected duplicate selector` that came from more shared styles.